src/index.js

```diff
@@ -20,10 +20,16 @@ export default function supernova(env) {
     sense,
     translator,
     flags: { isEnabled },
+    anything,
   } = env;
-  const senseNavigation = sense && sense.navigation;
+  const senseNavigation = sense?.navigation;
+  const isFeatureBlacklisted = anything.sense?.isFeatureBlacklisted;
+  const isUnsupportedFeature = anything.sense?.isUnsupportedFeature;
+  const shouldHide = (feature) =>
```

Comment:
I don't think it's a good idea to batch the hide condition like this. There is a risk, albeit a small risk, that this can produce a false positive. For example if a blacklisted feature is check with `isFeatureBlacklisted('bookmarks')` and returns false but `isUnsupportedFeature('bookmarks')` return true.

Reply:
I do not see that it produces a false positive. In your example, the shouldHide is true which looks correct to me. Did I miss anything?

Reply:
My assessment was based off that it should only hide given a specific scenario. So for example the apply bookmarks actions should ONLY hide given it's a blacklisted feature. So it should not matter if it's a disabled feature flag or unsupported feature.

Does that make sense?

Reply:
Make sense. Thx!